src/components/engine/configuration/system.tsx

```diff
@@ -16,6 +20,10 @@ export const EngineSystem: React.FC<EngineSystemProps> = ({ instanceUrl }) => {
       Version: {data.engineVersion ?? "..."}
       <br />
       Enabled: {data.features?.join(", ")}
+      <br />
+      CPU: {metrics.data?.data?.cpu ?? "..."}
```

Comment:
Lets limit the # decimals:
```suggestion
CPU: {metrics.data?.data?.cpu?.toFixed(1) ?? "..."}
```